### ISSUE TITLE
Alternating row darkness

### ISSUE THREAD

**Issue by 3C-Coleslaw** [swifty-report-cards/swifty-bot#9]
**Is your feature request related to a problem? Please describe.**
Hard to follow rows across the report.

**Describe the solution you'd like**
An easy fix would be to make alternating rows have a tad darker background. I'll attach an example screenshot.

```
name: 'new-command',
description: 'The command deals with ...',
isRestricted: false,
params: [
  {
    name: 'param1',
    validationTypes: [
      ['Number', 'The parameter has to be a number'],
    ],
  },
],
```
For `validationTypes`, it doesn't matter how you describe your validation method as long as it's accurate

**Describe alternatives you've considered**

**Additional context**
![default-table-style](https://user-images.githubusercontent.com/11131406/56606933-f6ceea00-65cc-11e9-95e2-d0b4300401a6.png)

**Comment by WebDeg-Brian** [comment]
Mind creating a PR with the enhanced report template?

**Comment by 3C-Coleslaw** [comment]
@WebDeg-Brian I'm not sure what you're asking.. lol What's a PR?
